content/en/docs/tasks/administer-cluster/running-cloud-controller.md

```diff
@@ -53,6 +53,13 @@ Successfully running cloud-controller-manager requires some changes to your clus
   kubelet is aware that it must be initialized by the cloud controller manager
   before it is scheduled any work.
 
+  {{< note >}}
```

Comment:
It might be clearer to just update the first point above to say:
```
`kube-apiserver` and `kube-controller-manager` must run with the `--cloud-provider=external`
  flag. This ensures that it does not run any cloud specific loops that would be run by
  cloud controller manager.
```

Reply:
i tend to agree, we can come back and change this once everything is deprecated/removed.

Reply:
@andrewsykim, @elmiko,
Sure, agree. Let me update it.

Reply:
> If you are reading this paragraph, and then the first bullet in the list, you may get confused. Agree?

Yeah, Agree on this point. it feels me also confused a bit.